doc/changelog/07-vernac-commands-and-options/15274-tacopt.rst

```diff
@@ -0,0 +1,8 @@
+- **Changed:** commands setting tactic options (currently
+  :opt:`Firstorder Solver` and :cmd:`Obligation Tactic`, as well as any
+  defined by third party plugins) now support :attr:`export` locality.
+  Note that the meaning of :attr:`global` and the default outside
+  sections currently also set the option when importing any
+  surrounding module. This will change in a future version. (`#15274
```

Comment:
```suggestion
  Note that :attr:`global` and the default outside
  sections are currently applied when importing any
  surrounding module. This will change in a future version. (`#15274
```

The "surrounding module" part doesn't make immediate sense to me.  If this is already in the doc in a reasonable place, is it necessary to tersly explain it again here?

Reply:
In 
~~~coq
Module M.
Module N.
End N.
End M.
~~~
M is a surrounding module for N

Reply:
could also be called parent module

Reply:
How about "Settings using `global` or without a locality attribute are currently applied when any module containing it (recursively) is imported."?